app/services/users.js

```diff
@@ -22,6 +25,12 @@ module.exports.signUp = body =>
     }
   });
 
+module.exports.index = page =>
+  models.user.findAndCountAll({ offset: numberOfRecords * page, limit: numberOfRecords }).catch(error => {
+    logger.error(error);
+    return error;
```

Comment:
Returning the error will make the flow continue normally. You should throw the error. Also, please remember to wrap it into our errors format.